packages/backend-common/src/reading/AzureUrlReader.ts

```diff
@@ -113,7 +119,7 @@ export class AzureUrlReader implements UrlReader {
 
     const commitsAzureResponse = await fetch(
       getAzureCommitsUrl(url),
-      await getAzureRequestOptions(this.integration.config),
+      await this.provider.getCredentials({ url: url }),
```

Comment:
Thinking it's best to explicitly pass just `headers` here, just in case

Reply:
Done 👍